### PR TITLE
Add missing minReadySeconds & strategy chart values in Helm Installation documentation

### PR DESCRIPTION
**Pull request by aknot242**
### Proposed changes
Adds missing minReadySeconds & strategy helm chart values reported in #2852 

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [X] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that all unit tests pass after adding my changes
- [X] I have updated necessary documentation
- [ ] I have rebased my branch onto main
- [X] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

File: docs/content/installation/installation-with-helm.md [hunk 1]
```diff
@@ -230,6 +230,8 @@ The following tables lists the configurable parameters of the NGINX Ingress Cont
 |``controller.readyStatus.enable`` | Enables the readiness endpoint `"/nginx-ready"`. The endpoint returns a success code when NGINX has loaded all the config after the startup. This also configures a readiness probe for the Ingress Controller pods that uses the readiness endpoint. | true |
 |``controller.readyStatus.port`` | The HTTP port for the readiness endpoint. | 8081 |
 |``controller.enableLatencyMetrics`` | Enable collection of latency metrics for upstreams. Requires ``prometheus.create``. | false |
+``controller.minReadySeconds`` | Specifies the minimum number of seconds for which a newly created Pod should be ready without any of its containers crashing, for it to be considered available. [docs](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#min-ready-seconds) | 0 |
+``controller.strategy`` | Specifies the strategy used to replace old Pods by new ones. [docs](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) | {} |
```
Comment (lucacome) [inline docs/content/installation/installation-with-helm.md:234]: Looks like there are some `|` missing at the beginning of the lines

**Review by lucacome — CHANGES_REQUESTED** [review]

File: docs/content/installation/installation-with-helm.md (hunk 1 above)
Comment (aknot242) [reply docs/content/installation/installation-with-helm.md]: Thanks. Added.

**Review by ciarams87 — APPROVED** [review]

File: docs/content/installation/installation-with-helm.md [hunk 2]
```diff
@@ -230,6 +230,8 @@ The following tables lists the configurable parameters of the NGINX Ingress Cont
 |``controller.readyStatus.enable`` | Enables the readiness endpoint `"/nginx-ready"`. The endpoint returns a success code when NGINX has loaded all the config after the startup. This also configures a readiness probe for the Ingress Controller pods that uses the readiness endpoint. | true |
 |``controller.readyStatus.port`` | The HTTP port for the readiness endpoint. | 8081 |
 |``controller.enableLatencyMetrics`` | Enable collection of latency metrics for upstreams. Requires ``prometheus.create``. | false |
+|``controller.minReadySeconds`` | Specifies the minimum number of seconds for which a newly created Pod should be ready without any of its containers crashing, for it to be considered available. [docs](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#min-ready-seconds) | 0 |
```
Comment (jputrino) [inline docs/content/installation/installation-with-helm.md:233]:
```suggestion
|``controller.minReadySeconds`` | Specifies the minimum number of seconds for which a newly created Pod should be ready, without any of its containers crashing, for it to be considered available. [docs](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#min-ready-seconds) | 0 |
```

File: docs/content/installation/installation-with-helm.md [hunk 3]
```diff
@@ -230,6 +230,8 @@ The following tables lists the configurable parameters of the NGINX Ingress Cont
...
+|``controller.strategy`` | Specifies the strategy used to replace old Pods by new ones. [docs](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) | {} |
```
Comment (jputrino) [inline docs/content/installation/installation-with-helm.md:234]:
```suggestion
|``controller.strategy`` | Specifies the strategy used to replace old Pods with new ones. [docs](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) | {} |
```

**Pull request merged**